source/MaterialXRenderGlsl/GLTextureHandler.cpp

```diff
@@ -62,6 +62,11 @@ bool GLTextureHandler::acquireImage(const FilePath& filePath,
         _boundTextureLocations.resize(maxTextureUnits, MaterialX::GlslProgram::UNDEFINED_OPENGL_RESOURCE_ID);
     }
 
+    if (filePath.isEmpty())
```

Comment:
Changes to this file look like an unintended reversion of an earlier fix in master.

Reply:
Looks like we didn't merge this back properly to our repo. Will revert.